docs/course_materials/exercises/check_helper.py

```diff
@@ -0,0 +1,91 @@
+import os
+from pathlib import Path
+import datetime as dt
+
```

Comment:
this exercise will run from the root but it should run on /home/sutudent. any chance you can add an option to change the home directory?

Reply:
Added as an option for the run command